apps/lfx/src/app/shared/components/button/button.html

```
@@ -0,0 +1,27 @@
+<!-- Copyright The Linux Foundation and each contributor to LFX. -->
+<!-- SPDX-License-Identifier: MIT -->
+
+<button
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)